help/user-guide/isajson-templates.md

```diff
@@ -0,0 +1,39 @@
+---
+title: SEEK User Guide - Experiment Sample Templates
+layout: page
+---
+
```

Comment:
I suggest to collect all information about the Experiment Sample Templates here and to link to this page from the page about ISA-JSON compliant experiments.